openassessment/templates/openassessmentblock/self/oa_self_assessment.html

```diff
@@ -45,17 +45,7 @@ <h3 class="self-assessment__display__title">{% trans "Your Response" %}</h3>
 
                     {% include "openassessmentblock/oa_submission_answer.html" with answer=self_submission.answer answer_text_label="Your response to the question above:" %}
 
-                    {% if allow_file_upload and self_file_url %}
-                    <header class="self-assessment__display__header">
-                        <h3 class="self-assessment__display__title">
-                            {% trans "Associated Image" %}
-                        </h3>
-                    </header>
-
-                    <div class="self-assessment__display__image">
-                        <img class="submission--image" alt="{% trans "The image associated with your submission." %}" src="{{ self_file_url }}"/>
-                    </div>
-                    {% endif %}
+                    {% include "openassessmentblock/oa_uploaded_file.html" with file_upload_type=file_upload_type file_url=file_url header="Associated File" class_prefix="self-assessment" %}
```

Comment:
I'm wondering how you ended up with `file_url=file_url`. I think you are suppose to pass `file_url=self_file_url`